W-15125370-Mentioning possible cipher suite preference change depending on JDK version. (#224)
mulesoft/docs-general: modules/ROOT/pages/java-support.adoc

* If your workloads are running on Runtime Fabric, you must allocate 1400 MB of memory for 0.50 vCPU replica sizes. Memory requirements vary for individual apps. 
* If your workloads are running on-premises, such as in a hybrid or standalone environment, you must tune your operating system's garbage collection algorithm. 

[[default-tls-cipher-selection]]
==== Default TLS Cipher Selection

> When using TLS in a server with Java 11 and later, the server is the one that determines the cipher preference. The client supported ciphers list is only used to filter, but never to determine priority. See https://bugs.openjdk.org/browse/JDK-8168261[Use server cipher suites preference by default (JDK-8168261)].

Reviewer: ```suggestion
When using TLS in a server with Java 11 and later, the server determines the cipher preference. The client supported ciphers list is used to filter, not to determine priority. See https://bugs.openjdk.org/browse/JDK-8168261[Use server cipher suites preference by default (JDK-8168261)].
```

Reviewer: These sentences seemed to have too much emphasis, so I made them more neutral.

> Upgrading your server from Java 8 to Java 11 and later may cause a different cipher to be negotiated (depending on the supported ciphers list of the client). A change in cipher may incur in a performance vs security trade-off.

Reviewer: ```suggestion
Upgrading your server from Java 8 to Java 11 and later can cause a different cipher to be negotiated (depending on the supported ciphers list of the client). A change in cipher can affect its performance and security.
```

> If you want to change the defaults, see xref:mule-runtime::tls-configuration.adoc#optional-specify-protocols-and-cipher-suites[control the cipher selection].

Reviewer: ```suggestion
If you want to change its default values, see xref:mule-runtime::tls-configuration.adoc#optional-specify-protocols-and-cipher-suites[control the cipher selection].
```
